sources/niUI/src/FontTTF.cpp

```diff
@@ -23,7 +23,7 @@ static sPropertyBool _dumpGlyphs("niUI.FontTTF.DumpGlyphs",eFalse);
 
 #define TTF_KERNING
 // #define TTF_PREMULALPHA
-#define TTF_PAGE_SIZE 512
+#define TTF_PAGE_SIZE 1024
```

Comment:
I'd rather keep it at 512 by default. Maybe better if we make this a property of FontTTF and make it configurable at runtime?